meals.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const knex = require('knex')({
```

Comment:
Looks like duplicate code in both index.js and this file, find a way to not repeat yourself :) The reason being that if you need to change the connection code in the future you would need to change it two places.